docs/getting-started/quickstart.md

```diff
@@ -9,7 +26,7 @@ In this tutorial we will
 5. Read the latest features from the online store for real-time inference.
 6. Explore the (experimental) Feast UI
 
-## Overview
+***Note*** - Feast can used as an executable or as a server, please refer to [feast feature server](../reference/feature-servers/python-feature-server.md)
```

Comment:
Would re-phrase:

"Feast provides both a python SDK as well as an optional hosted service for reading and writing feature data to the online and offline data stores. The latter might be useful when non-python languages are required.

 For this tutorial, we will be using the python SDK."

Reply:
Done